hw2/03-form.js

```diff
@@ -1,3 +1,77 @@
-/** Exercise 03 - Form **/
+/** 
+ * Exercise 03 - Form 
+ * Brooke Czerwinski
+ * Full-Stack Web Development
+ * HW 2
+ * 
+ * References:
+ * https://getbootstrap.com/docs/5.0/forms/overview/
+ * https://getbootstrap.com/docs/5.0/forms/form-control/
+ * https://getbootstrap.com/docs/5.0/forms/checks-radios/
+ * https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset
+ * https://developer.mozilla.org/en-US/docs/Web/API/FormData/values
+ * https://getbootstrap.com/docs/5.0/forms/validation/
+ * https://stackoverflow.com/questions/20573488/why-does-html5-form-validation-allow-emails-without-a-dot
+ * **/
 
-// Add your code here
+// Form id is passed in so it can be accessed
+const resetForm = (formID) => {
+    // Reset the form
+    document.getElementById(formID).reset();
+}
+
+const validateSignup = () => {
+    // Get info from form
+    const inputName = document.getElementById("inputName").value;
+    const inputEmail = document.getElementById("inputEmail").value;
+    const inputFeedback = document.getElementById("inputFeedback").value;
+    const inputNewsletter = document.getElementById("inputNewsletter").checked;
+
+    // Check if feedback was submitted to build feedback output string
+    let feedbackStr = ``
+    if (inputFeedback !== ``) {
+        feedbackStr = inputFeedback;
+    }
+    else {
+        feedbackStr = 'No feedback was submitted.';
+    }
+
+    // Check newsletter value to build newsletter output string
+    let newsletterStr = ``
+    if (inputNewsletter) {
+        newsletterStr = `Yes, I would like to join the newsletter.`;
+    }
+    else {
+        newsletterStr = `No, thank you.`;
+    }
+
+    // Print message to the console
+    console.log(`========= Form Submission =========
```

Comment:
not the right way for console.log

Reply:
Thank you. Are you able to clarify why you think it is not the right way? Console.log can contain multiline outputs when using the backtick character. The output matches exactly as the professor requested in the assignment. So I am unable to understand why you request a change here.

Reply:
Actually, after thinking about it for a little while, I decided it was probably better to build the output as a string and then output the string to `console.log()`. I'm not sure if that's what you meant, but if so, good catch!

Reply:
grouping has to be used see the chapter as mentioned by professor

Reply:
Thank you! I completely forgot about `console.group()`. I pushed another commit where I fixed it. Thank you for pointing this out.